lib/matplotlib/font_manager.py

```diff
@@ -212,6 +213,7 @@ def win32FontDirectory():
         return os.path.join(os.environ['WINDIR'], 'Fonts')
 
 
+@_api.deprecated("3.5")
 def _win32RegistryFonts(reg_domain, base_dir):
```

Comment:
This is private, so unless it was exposed somewhere, you can just delete it.

Reply:
this is just used in _win32InstalledFonts; the point of marking it as deprecated is just so that we don't forget to also remove it in 3.7.

Reply:
But that's deprecated too; won't that cause doubled warnings, one for each?

Reply:
fair point, I replaced the deprecation by a comment in win32InstalledFonts.